src/components/theme/Footer/Footer.jsx

```diff
@@ -79,8 +80,10 @@ const Footer = ({ intl }) => (
         <List.Item href="/accessibility-info">
           <FormattedMessage id="Accessibility" defaultMessage="Accessibility" />
         </List.Item>
-        <List.Item href="/contact">
-          <FormattedMessage id="Contact" defaultMessage="Contact" />
+        <List.Item>
+          <Link to="contact-info" className="item">
```

Comment:
@cekk Maybe this is our chance here to rename "/contact-info" to something more sane. This is a contact form, right? I never understood why we use "info" here. Especially since Plone does not allow you to edit the contact info page as an editor.

Reply:
Yes, you're right.
I had a discussion with @robgietema in Tokyo and we decided to keep "contact-info" for retro-compatibility with the plone environment.
But i'd like to change it in "contact" or "contacts" or whatever.
When we decide the name, i could also fix the typo in the other comment.

P.S.: github didn't emailed me yesterday with this review :(

Reply:
@sneridagh @robgietema I'd prefer to change "contact-info" to "contact-form" but I won't fight for it. My opinioin is that if we don't change things now in Volto for the better, we never will. :)

Reply:
contact-form is fine for me also

Reply:
But then we also need to change the classname etc to make in consistant.